sdk/streamlib-python-wheel/tests/test_native_builtin_blocks.py

```
@@ -0,0 +1,180 @@
+import threading
+
+import pytest
+
+import streamlib
+# `input` is streamlib's port decorator — the test reads like user code,
+# which spells it exactly this way.
+from streamlib import (  # noqa: A004
+    RuntimeContextLimitedAccess,
+    TestPatternSource,
+    VideoFrame,
+    input,
```

Ruff (0.16.1)

[error] Import `input` is shadowing a Python builtin

(A004)